independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/AnnotationLiteralGenerator.java

```diff
@@ -71,7 +71,8 @@
 
     void createSharedAnnotationLiteral(ClassOutput classOutput, Key key, Literal literal, Set<String> existingClasses) {
         // Ljavax/enterprise/util/AnnotationLiteral<Lcom/foo/MyQualifier;>;Lcom/foo/MyQualifier;
-        String signature = String.format("Ljavax/enterprise/util/AnnotationLiteral<L%1$s;>;L%1$s;",
+        String signature = String.format("L%1$s<L%2$s;>;L%2$s;",
+                AnnotationLiteral.class.getName().replace('.', '/'),
```

Comment:
FTR in JDK 12+ `java.lang.Class#descriptorString` can be called instead

Reply:
But we're still on JDK 11, or?

Reply:
Yes, still in JDK11, just a curiosity 😉